client/src/extension.ts

```diff
@@ -143,3 +126,31 @@ function getProbeLocations(configName: string, bundled: string): string[] {
   locations.push(bundled);
   return locations;
 }
+
+/**
+ * Construct the arguments that's used to spawn the server process.
+ * @param ctx vscode extension context
+ * @param debug true if debug mode is on
+ */
+function constructArgs(ctx: vscode.ExtensionContext, debug: boolean): string[] {
+  const config = vscode.workspace.getConfiguration();
+  const args: string[] = [];
+
+  const ngLog = config.get('angular.log') as string;
```

Comment:
Is the return value of `get` typed? If not, prefer pre-assignment typing:
```suggestion
  const ngLog: string = config.get('angular.log');
```

Reply:
Good call! Your suggestion is what ts-style guide recommends. Thanks!